user-guide/Advanced_Functionality/Databases/Supported_System_Data_Storage_Architectures/Resource_migration_to_elastic.md

```diff
@@ -0,0 +1,139 @@
+Field names in Elasticsearch have a maximum length of 32766 bytes, which means any field of a resource or resource pool indexed in Elasticsearch only contain 32766 bytes. This limit is mostly important for string fields, which can contain 32766 characters of one byte (or 16383 characters of two bytes). When trying to save a resource or resource pool with a field that is too big, the API will return an ``UnknownError``. The ``SLResourceManager.txt`` log file will contain the actual exception, which will mention the field that could not be indexed in Elasticsearch.
+
+### Number of indices
+
+When a resource is indexed in elasticsearch, all its properties, capacities and capabilities will be indexed as well. This means each unique property name and unique capacity and capability id will become a field mapping in Elasticsearch. If the index contains a lot of such mappings, Elasticsearch starts having performance problems. During testing, no issues were found when using a typical amount of capacities, capabilities and property names, but slowdowns started to be noticeable when more than 300 unique field mappings were present.
+
+### Difference when deleting resources
+
+When using ``Xml`` as storage, it is not allowed to remove a resource while one of the agents in the cluster is unreachable, as this could cause syncing issues. When using Elasticsearch as storage, this is allowed.
+
+### Initial event on subscriptions
+
+When using ``Xml`` as storage, if a client subscribes on the ``ResourceManagerEventMessage``, he will receive an initial event with all resources and resource pools. When using Elasticsearch as storage these events will no longer be sent. This is to prevent sending a message to the client containing thousands of resources.
+
+### Syncing
+
+When using ``Xml`` storage, all ResourceManagers in the cluster will sync the resources in their ``Xml`` file on startup and during the midnight sync. When using Elasticsearch as storage, DataMiner relies on Elasticsearch to do the syncing and data will not be sent around during the midnight sync or during startup. The ResourceManager will refresh the cached resources during the midnight sync by reading all resources and resource pools again from Elasticsearch.
+
+## Metrics
+
+### Resource performance
+
+Below you can find the result of our performance testing for resource add, update and delete operations. On our setup (with a local Elasticsearch), Elasticsearch started to outperform ``Xml`` storage when more than 2000 resources were present on the system.
+
+![SRM Resource create performance](~/user-guide/images/SRM_Resource_Create_Performance.png)
+![SRM Resource update performance](~/user-guide/images/SRM_Resource_Update_Performance.png)
+![SRM Resource delete performance](~/user-guide/images/SRM_Resource_Delete_Performance.png)
+
+__Note__: read performance of resources and resource pools is unchanged, since all reads will go to a cache in SLNet. This was also confirmed during our testing.
```

Comment:
```suggestion
> [!NOTE]
> Read performance of resources and resource pools is unchanged, since all reads will go to a cache in SLNet.
```

I'd prefer to leave out "This was also confirmed during our testing" as this has no added value. Users can naturally assume that we test things in order to confirm what we write here, so we don't need to explicitly state this.